impeller/aiks/aiks_unittests.cc

```diff
@@ -2877,6 +2877,35 @@ TEST_P(AiksTest, CanCanvasDrawPicture) {
   ASSERT_TRUE(OpenPlaygroundHere(canvas.EndRecordingAsPicture()));
 }
 
+TEST_P(AiksTest, DrawPictureWithText) {
```

Comment:
Sorta nit: The text seems to either be invisible or hard to see in the goldens with the golden service's white background. Maybe throw in black backgrounds with `DrawPaint`?

Reply:
Actually, seems like the new goldens aren't rendering anything for some reason.

Reply:
Did the gold test part have a chance to run?

The text is red, I thought that should render on either black or white background fine :\

Reply:
But before this change, the new tests would've had validation errors (the atlas would not contain the scale and the test would crash)